nilearn/decoding/decoder.py

```diff
@@ -1460,29 +1429,6 @@ def __sklearn_tags__(self):
         """
         return super().__sklearn_tags__()
 
-    @fill_doc
-    def fit(self, X, y, groups=None):
-        """Fit the decoder (learner).
```

Comment:
moved to mixin class

Reply:
Did this work as expected? The call to `super.fit` would run `_BaseDecoder.fit`. But `_BaseDecoder` is not a superclass of mixin.

Reply:
sorted in a coming PR